parsl/monitoring/monitoring.py

```diff
@@ -525,5 +533,13 @@ def monitor(pid,
         except Exception:
             logging.exception("Exception getting the resource usage. Not sending usage to Hub", exc_info=True)
 
+        try:
+            msg = command_q.get(block=False)
+            if msg == "Finished":
+                logging.info("Received task finished message. Ending the monitoring loop now.")
+                break
+        except Queue.empty:
```

Comment:
@ZhuozhaoLi should this be `queue.Empty`? (different capitalization)

Reply:
You are right. I fixed this in the new PR #1482